resources/sass/components/_map-aside.scss

```diff
@@ -100,23 +117,59 @@
     }
     &__subtitle {
         margin: $indent-ml 0 $indent-sm 0;
+        @media screen and (max-width: 1040px) {
+            display: none;
+        }
     }
     &__caption {
         color: $c-gray8;
+        @media screen and (max-width: 1040px) {
+            display: none;
+        }
     }
     &-theme-list {
         list-style-type: none;
         padding: 0;
         font-size: $fs-l;
+        $this: &;
+        @media screen and (max-width: 1040px) {
+            position: fixed;
+            top: 100%;
+            transform: translateY(-100%);
+            overflow-x: scroll;
+            overflow-y: hidden;
+            display: flex;
+            width: 100%;
+            z-index: 1000000;
```

Comment:
not necessary has be changed, but imo its better to keep z-index values as variables in a separate file, so at least you can see what exactly will be overlapped when you need to add another one or when you need to change existed values

Reply:
is fixed

Reply:
fixed